query-dispatcher/routes.js

```diff
@@ -6,8 +6,9 @@ const { handleAxiosError } = require('./utils');
 
 // const messenger = process.env.MESSENGER_URL ||
 //                    'http://robokop.renci.org:4868/answer';
-const strider = process.env.STRIDER_URL ||
-                    'http://robokop.renci.org:5781';
+// const strider = process.env.STRIDER_URL ||
+//                     'http://robokop.renci.org:5781';
+const robokop = process.env.ROBOKOP_URL || 'http://robokop.renci.org:7092';
```

Comment:
Is this referring to the external service RoboKop or is this for qgraph?

Reply:
This is the external Robokop ARA.